### DISCUSSION TITLE
Multi-Level Collections

### DISCUSSION THREAD

**Discussion by KalbazJarak** [gotson/komga#1544] [Ideas]
### Describe your suggested feature

It would be useful to be able to define multi-level collections as a way to organise comics in a more granular way, something usually done without Komga using nested folders.

For example you could have collections like this:

- Star Wars
	- Legends
		- New Republic
		- Movie Era
		- Post-Movies
	- Canon
		- Old Republic
		- New Republic
		- Movie Era
- Hellboy Universe
	- Hellboy
		- Mainline
		- Short Stories
	- BPRD
		- Mainline
		- Historical
	- Witchfinder

For large libraries this allows more structured organisation without having to change the existing series folder reading logic to accommodate multi-level folder structures, though SQLite doesn't allow nested tables and I'm not great with databases so not sure what the best solution here would be.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
Just add that in the name ?

**Comment by KalbazJarak** [comment]
While that does technically categorise stuff it doesn't really work as effective navigation. Rather than having 20 top-line collections say on 1 page that you can narrow down by clicking through you just end up with 500 spread across 25 pages. 

Maybe if you can remember everything you have in a giant library it sort of works but if you're sharing your library with multiple users it's not great UX.

**Comment by gotson** [comment]
> Maybe if you can remember everything you have in a giant library it sort of works but if you're sharing your library with multiple users it's not great UX.

that's where the search comes in handy :)

**Comment by KalbazJarak** [comment]
But that's still making the assumption they roughly know what they want, that doesn't help if they're just browsing to look for new things. Sure if they're familiar with superheroes they can search Batman but publishers like Image or Boom put out loads of different stuff, so in that case you could make multi-level collections that go Publisher -> Genre, or the other way around, or sub-genres for them to flick through.

**Comment by Cantello** [comment]
Could be probably done with smart lists, tags & volumes, see #200.

**Comment by gotson** [comment]
Similar to #1534

**Comment by Blazeflack** [comment]
I think this feature could be handy.
